app/src/main/cpp/jniWallet.cpp

```diff
@@ -515,9 +517,13 @@ JNIEXPORT void JNICALL
 Java_com_tari_android_wallet_ffi_FFIWallet_jniLogMessage(
         JNIEnv *jEnv,
         jobject jThis,
-        jstring jMessage) {
+        jstring jMessage,
+        jobject error) {
+    int i = 0;
+    int *r = &i;
```

Comment:
Consider change names of these values. `errorCode` and `errorCodePointer` should be ok.

Reply:
like that idea, but should do it on whole wrapper